tests/integration/learning/test_domains.py

```diff
@@ -124,6 +131,7 @@ def test_learner_ignores_stored_nodes_from_other_domains(
     assert pest not in other_staker.known_nodes  # But not anymore.
 
 
+@pytest.mark.skip
```

Comment:
What is the reason for skipping these tests? I assume it's temporary?

Reply:
Ah yes, thanks for the reminder, I'll attempt to restore these tests before merging.